Description: fix library version
 This patch increases library version to 1.0.0 due to
 removed interfaces in fso-specs.
Author: Rico Rommel <rico@bierrommel.de>

---
Forwarded: no
Last-Update: 2012-03-10

--- libfso-glib-2012.02.16.1.orig/src/Makefile.am
+++ libfso-glib-2012.02.16.1/src/Makefile.am
@@ -56,6 +56,8 @@ libfso_glib_la.vala.stamp: gen-bindings
 	$(VALAC) -C ${VALACFLAGS} --basedir . $(libfso_glib_la_VALASOURCES)
 	touch $@
 
+libfso_glib_la_LDFLAGS = -version-info 1:0:0
+
 gen-bindings:
 	vala-dbus-binding-tool $(DBUSBINDINGTOOLFLAGS)
 
